proto/pdpb.proto

```diff
@@ -250,7 +250,11 @@ message BootstrappedError {
     
 }
 
+message StoreIsTombstoneError {
+}
+
 message Error {
-    optional string message                 = 1;
-    optional BootstrappedError bootstrapped = 2;
+    optional string message                     = 1;
+    optional BootstrappedError     bootstrapped = 2;
```

Comment:
I don't think so, this should be a `BootstrappedError` message, not a `Bootstrapped` message.